DataGrid: add note about editing mode (T1123375) (#4703)
DevExpress/devextreme-documentation: concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md

> For example, you can specify the [items](/api-reference/10%20UI%20Components/dxForm/1%20Configuration/items.md '/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items') array to reorder editable fields (or "simple items", as they are called in the Form UI component), or organize them in groups and tabs.

Reviewer: If a group is preferable, move it to the first position: group, tabs, or leave items ungrouped.

> [note] We recommend that you wrap all items in groups to ensure that the form fields works properly.

Reviewer 2: ```suggestion
[note] We recommend that you wrap all items in groups to ensure that the form fields work properly.
```
What does "properly" mean? Not clear.

Author: Please, refer to the following ticket: https://supportcenter.devexpress.com/internal/ticket/details/T1123375

Reviewer: You can just leave the recommendation without explanation.

```suggestion
[note] The recommended way is to wrap all items in groups.
```
